index.html

```diff
@@ -886,21 +890,14 @@ <h2>
         on a checkbox, but updates the checkbox checked state in case the wake
         lock state changes:
       </p>
-      <pre class="example js">
+      <pre class="example js" id="onrelease-example">
         const checkbox = document.createElement("input");
         checkbox.setAttribute("type", "checkbox");
+        checkbox.checked = true;
```

Comment:
Maybe this line should move down to after the lock is acquired? Otherwise the checkbox is checked too early.

Reply:
It makes sense, I've done that in the new patch version, thanks.